src/logistration/RegistrationPage.jsx

```diff
@@ -40,6 +43,7 @@ class RegistrationPage extends React.Component {
 
   handleSubmit = (e) => {
     e.preventDefault();
+    const params = getQueryParameters();
```

Comment:
It's deprecated, instead use `URL.searchParam`, check this for more details https://developer.mozilla.org/en-US/docs/Web/API/URL/searchParams. Also replace it in `LoginPage.jsx`.

Reply:
I believe you forgot to change it here?